app/views/books/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<h1 class="top-title align-center">Media Ranker Books</h1>
+
+<main class="main-container">
+  <article class="table-headers row">
+    <h3 class="small-12 medium-4 column">Title: </h3>
+    <h3 class="small-12 medium-4 column">Author: </h3>
+    <h3 class="small-12 medium-2 column">Rank: </h3>
+    <h3 class="small-12 medium-1 column">Vote: </h3>
+  </article>
+  <div class="media-list-container">
+  <% @books.each do |book| %>
+    <article class="one-book row">
+      <h5 class="small-12 medium-4 column"><%= link_to book.title, book_path(book.id) %></h5>
+      <h5 class="small-4 medium-4 column"><%= link_to book.author, book_path(book.id) %> </h5>
+      <p class="small-7 medium-2 column right-padding"> Votes: <%= book.rank %></p>
+      <div class="small-6 medium-1 column arrows">
+        <p class="upvote-arrow"><%= link_to "▲  ", {controller: 'books', action: 'upvote', id: book.id}, method: :patch%></p>
```

Comment:
Since we have a named route for this, we can also use `upvote_book_path(book.id)`.